package.json

```diff
@@ -27,18 +27,18 @@
     "email": "peter.han.9493@gmail.com",
     "url": "https://github.com/peterthehan"
   },
-  "main": "dist/index.js",
-  "bin": "dist/index.js",
+  "main": "src/index.js",
+  "bin": "src/index.js",
   "scripts": {
     "action:build": "npx tsc --noEmit",
     "action:check": "prettier --no-config --check .",
     "action:lint": "npx eslint .",
     "bot:dev": "nodemon app/src/index.js",
     "bot:start": "node app/src/index.js",
     "build": "npx tsc",
```

Comment:
do we need this `build` script now that we're not compiling?

Reply:
and if we don't need this, can you also remove line 54 in CONTRIBUTING.md that docs what this `build` script does